plugins/pencilblue/controllers/actions/admin/sites/new_site.js

```diff
@@ -0,0 +1,82 @@
+/*
+ Copyright (C) 2015  PencilBlue, LLC
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+module.exports = function NewSiteActionModule(pb) {
+
+    //pb dependencies
+    var util = pb.util;
+
+    /**
+     * Creates a new site
+     */
+    function NewSiteAction(){}
+    util.inherits(NewSiteAction, pb.BaseController);
+
+    NewSiteAction.prototype.render = function(cb)
+    {
+        var self = this;
+
+        this.getJSONPostParams(function(err, post) {
```

Comment:
specify request_body mime types in route definition to avoid having to make this call manually.